docs/core/migration/index.md

```diff
@@ -36,7 +36,7 @@ Files that were migrated (*project.json*, *global.json*, *.xproj* and solution f
 A file named *UpgradeLog.htm* is also saved and automatically opened that contains a migration report.
 
 > [!IMPORTANT]
-> The new tooling is not available in Visual Studio 2015, so you cannot migrate your projects using that version of Visual Studio.
+> The above tooling is only available between Visual Studio 2017 (version 15.0) and Visual Studio 2019 (version 16.3). If you are using a version of Visual Studio outside of that range, you should either install a suitable version of Visual Studio or use the command line migration tool described below.
```

Comment:
It won't be in 16.3 right? So, this should say 16.2?

Reply:
Thanks, fixed.